helm_deploy/hmpps-person-record/values.yaml

```diff
@@ -15,7 +15,7 @@ generic-service:
     host: app-hostname.local # override per environment
     tlsSecretName: hmpps-person-record-cert
     annotations:
-      nginx.ingress.kubernetes.io/configuration-snippet: |
+      nginx.ingress.kubernetes.io/server-snippet: |
```

Comment:
Happy to approve, but whats this change actually doing ?

Reply:
https://github.com/ministryofjustice/hmpps-helm-charts/releases/tag/generic-service-3.0.0 is all I have as a reference for this - I haven't gone into detail of what the difference is between these two annotations